src/Message/AbstractResponse.php

```diff
@@ -1,91 +1,108 @@
 <?php
 
+declare(strict_types=1);
+
 namespace Omnipay\Windcave\Message;
 
+use InvalidArgumentException;
+use Omnipay\Common\Message\AbstractResponse as CommonAbstractResponse;
+
 /**
  * Response class for all Windcave requests
  */
-class AbstractResponse extends \Omnipay\Common\Message\AbstractResponse
+class AbstractResponse extends CommonAbstractResponse
 {
-    /** @var string HTTP response code */
-    protected $httpResponseCode = null;
+    protected string $httpResponseCode;
```

Comment:
Any particular reason that this is a string? This is an int in response classes etc. could be good to be consistent with that.

Reply:
I deferred to the type in the original annotation for this one as I figured implementations would be providing strings.  On the `set` method I've allowed int and string input but cast to string.

I could do the reverse (default int, accept int|string, cast to string).  However if I do that I'll probably need to do the same in the westpac package and others that implement httpResponseCode.

Reply:
I guess it's not a big deal, but I've only ever seen response codes as ints, so ideally this would also follow this pattern.